pytalibstream/ema_keeper.py

```diff
@@ -0,0 +1,27 @@
+import pyslidingwindow
+
+
+class EmaKeeper:
+    def __init__(self, max_len: int):
+        self.arr = pyslidingwindow.SlidingWindowFloat(max_len)
+        self._ema = 0
+
+    def add(self, value: float):
+        if self.arr.length() + 1 < self.arr.max_len():
+            self.arr.push(value)
+        elif self.arr.length() + 1 == self.arr.max_len():
+            self.arr.push(value)
+            self._ema = sum(self.arr) / self.arr.length()
```

Comment:
I think your previous code was okay.
if there isn't enough value in the array, we should use sma.

and `self.arr.length()` will never exceed `self.arr.max_len()`

should be like this:
```
if self.arr.length() < self.arr.max_len():
            self.arr.push(value)
            self._ema = sum(self.arr) / self.arr.length()
```